CHANGELOG.md

```diff
@@ -12,6 +12,7 @@
   This is a **breaking change** and will likely result in query errors on rollout as the query signature b/n QueryFrontend & Querier has changed. [#557](https://github.com/grafana/tempo/pull/557)
 * [ENHANCEMENT] Add list compaction-summary command to tempo-cli [#588](https://github.com/grafana/tempo/pull/588)
 * [ENHANCEMENT] Add list and view index commands to tempo-cli [#611](https://github.com/grafana/tempo/pull/611)
+* [ENHANCEMENT] Add a configurable prefix for HTTP endpoints. [#631](https://github.com/grafana/tempo/pull/631)
```

Comment:
Does this count as a breaking change?

Reply:
Unsure. It technically unbreaks the hardcoded `/tempo` breaking change :). So 0.6.0 => 0.7.0 will not be a breaking change.  While this exact commit is a breaking change from the previous one.

I think for this reason we leave it out.